site/src/atomics/overflow.md

```diff
@@ -36,12 +38,63 @@ You can use `.overflow-x-hidden` to set the clipping behavior for the left/right
 
 ```html
 <div class="border-radius-lg">
-    <div class="">
-        <div class="overflow-x-hidden" style="width: 100px">
-          <p class="font-size-xl" style="width: 150px">
-            This text is clipped on its right edge because we gave the parent container overflow-hidden.
-          </p>
-        </div>
-    </div>
+	<div class="">
+		<div class="overflow-x-hidden" style="width: 100px">
+			<p class="font-size-xl" style="width: 150px">
+				This text is clipped on its right edge because we gave the parent container overflow-hidden.
+			</p>
+		</div>
+	</div>
 </div>
 ```
+
+## Scrollbar gutter
+
+When using automatic overflow, as with the [scrolling components](../components/scroll.md), the sudden appearance or disappearance of a container's scrollbar can cause the container's content to jump around. While this might be fine if the scrollbar appears or disappears as a result of the user zooming or resizing their browser, it can look clunky otherwise — such as when messages are appended to a constrained-width or constrained-height chat container, resulting in a scrollbar where there wasn't one previously.
```

Comment:
Not wholesale against this, but curious if this makes the scrollbars appear on platforms like mac/safari that hide them when they're inactive.

Reply:
[Safari doesn't support this](https://caniuse.com/mdn-css_properties_scrollbar-gutter) I suspect precisely because it uses overlay scrollbars.

But this would absolutely be worth further testing to be sure!

Reply:
Per [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter), when overlay scrollbars are used, a la macOS, there is no gutter, so this wouldn't have any impact.